### PR TITLE
feat(discord): Split import channel signals and community import signals

### PR DESCRIPTION
**Pull request by borismelnik**
Adds the new signals to import single discord channel to community:
- Import progress
- Finish
- Canceled 

Important changes:
- [x] Something worth noting for reviewers.

Closes #

### CODE REVIEW

File: protocol/messenger_config.go [hunk 1]
```diff
@@ -55,6 +55,9 @@ type MessengerSignalsHandler interface {
 	DiscordCommunityImportProgress(importProgress *discord.ImportProgress)
 	DiscordCommunityImportFinished(communityID string)
 	DiscordCommunityImportCancelled(communityID string)
+	DiscordChannelImportProgress(importProgress *discord.ImportProgress)
+	DiscordChannelImportFinished(channelID string)
+	DiscordChannelImportCancelled(communityID string)
```
Comment (jrainville) [inline protocol/messenger_config.go]: should this be `channelID`?

File: signal/events_discord_import.go [hunk 2]
```diff
@@ -21,6 +21,18 @@ const (
 	// EventDiscordCommunityImportCancelled triggered when importing
 	// the discord community was cancelled
 	EventDiscordCommunityImportCancelled = "community.discordCommunityImportCancelled"
+
+	// EventDiscordChannelImportProgress is triggered during the import
+	// of a discord community channel as it progresses
+	EventDiscordChannelImportProgress = "community.discordChannelImportProgress"
+
+	// EventDiscordChannelImportFinished triggered when importing
+	// the discord community into status was successful
```
Comment (jrainville) [inline signal/events_discord_import.go]:
```suggestion
	// the discord community channel into status was successful
```

File: signal/events_discord_import.go [hunk 3]
```diff
@@ -21,6 +21,18 @@ const (
...
+	EventDiscordChannelImportFinished = "community.discordChannelImportFinished"
+
+	// EventDiscordChannelImportCancelled triggered when importing
+	// the discord community was cancelled
```
Comment (jrainville) [inline signal/events_discord_import.go]:
```suggestion
	// the discord community channel was cancelled
```

**Review by jrainville — COMMENTED** [review]
Looks good, just possible "typos"

**Review by saledjenic — APPROVED** [review]

**Review by jrainville — APPROVED** [review]

File: protocol/messenger_communities.go [hunk 4]
```diff
@@ -4277,15 +4286,16 @@ func (m *Messenger) RequestImportDiscordChannel(request *requests.ImportDiscordC
 					importProgress.AddTaskError(discord.ImportMessagesTask, discord.Error(err.Error()))
 					importProgress.StopTask(discord.ImportMessagesTask)
 					progressUpdates <- importProgress
-					cancel <- request.DiscordChannelID
+					cancel <- []string{string(request.CommunityID), request.DiscordChannelID}
+					fmt.Println("LALALA save messages failed:", err.Error())
```
Comment (jrainville) [inline protocol/messenger_communities.go]: leftover

**Review by jrainville — APPROVED** [review]

**Pull request merged**